php/FlatbufferBuilder.php

```diff
@@ -593,6 +593,10 @@ public function endVector()
 
     protected function is_utf8($bytes)
     {
+        if (function_exists('mb_detect_encoding') && (bool) mb_detect_encoding($bytes, 'UTF-8', true)) {
```

Comment:
my suggestion was to write `return (bool) mb_detect_encoding($bytes, 'UTF-8', true)` inside the `if`, because the way you've written it now, if mb_detect_encoding returns false, it is still going run the code underneath, which is not necessary.

Also, maybe it is useful to add a test case for this function, one with and one without utf-8 data?